src/UI/Implementation/Component/Toast/Factory.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
Please add `strict_types` here (or in the related PR).

Reply:
Resolved in dependend PR https://github.com/ILIAS-eLearning/ILIAS/pull/3671